examples/simple-generator/README.adoc

```diff
@@ -0,0 +1,26 @@
+# Example: Simple Yeoman Generator For Fiori Elements
+A simplified example illustrating how to create a yeoman generator using the `fiori-elements-writer` and the `axios-extension`.
+
+## Example
+This example project contains a generator that asks a few simple questions and then uses the `@sap-ux/fiori-elements-writer` to generate a Fiori application. Additionally it generates a karma config for the project utilzing our new mockserver middleware.
+
+## Prerequisite
+Before you can use the generator, you need to globally install `yo` and build the generator
+```
+npm install -g yo
+pnpm build
```

Comment:
```suggestion
npm build
```

Reply:
Well, the description is how to use it here, and there you require `pnpm build`, it also assumes that you did a `pnpm install` for the repo before.